src/main/java/Main.java

```diff
@@ -112,22 +187,35 @@ private static void waitForGobPieConf() {
         try {
             watchService = FileSystems.getDefault().newWatchService();
             Path path = Paths.get(System.getProperty("user.dir"));
-            path.register(watchService, StandardWatchEventKinds.ENTRY_CREATE, StandardWatchEventKinds.ENTRY_MODIFY);
+            path.register(watchService, StandardWatchEventKinds.ENTRY_MODIFY);
```

Comment:
Does this still account for file creation though?

Reply:
I removed it because there is a chance that one creates an empty `gobpie.json` file and in that case, it proceeds and runs into an exception (nullpointer) trying to parse JSON from an empty file. But when one "creates" a file with the configurations already in it, it is considered a modification and the code continues.